frontend/src/js/components/devices/AuthorizedDevices.test.tsx

```diff
@@ -40,7 +40,7 @@ const preloadedState = {
 describe('AuthorizedDevices Component', () => {
   it('renders correctly', async () => {
     const { baseElement } = render(<Authorized states={routes} />, { preloadedState });
-    const view = baseElement.firstChild;
+    const view = baseElement;
```

Comment:
just out of curiosity: was there anything missing in the snapshot?

Reply:
would render this: https://gitlab.com/Northern.tech/Mender/mender-server/-/jobs/8786463870#L5452